rasgoudt/requirements.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
You don't need a requirements.txt and a setup.py, it just creates two places where people can screw up adding package dependencies (like I did)

I recommend removing this file and using `pip install .` if you want to install the package locally during testing and whatnot

Reply:
we use the `requirements.txt` filte to import dependencies in the docker container while building/publishing the dist. We may not want to keep the docker deployment process here. I kept it bc that's what we do in pyrasgo and figured we should debate a change as a team. I'll bring this topic to standup PL tomorrow.

Reply:
per parkinglot, we now import the reqs in the setup.py file from this file so we'll only need to make the change in one place :D